src/usr/local/www/system_certmanager.php

```diff
@@ -1223,7 +1223,6 @@ function list_csrs() {
 						if (cert_get_ocspstaple($cert['crt'])) {
 							$certextinfo .= '<b>' . gettext("OCSP: ") . '</b> ';
 							$certextinfo .= gettext("Must Staple");
-						}
```

Comment:
Why did this brace get deleted?

Reply:
When I rebased my repo on the main one, It created a duplicate of the code block. I hit 'd5d' in vi instead of 'd4d'. fixed